packages/app/src/hooks.ts

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useState } from 'react';
+
+export function useTimeout(ms?: number) {
+  const [isReady, setReady] = useState(false);
+
+  useEffect(() => {
+    const id = setTimeout(() => setReady(true), ms);
+
+    return () => clearTimeout(id);
+  });
```

Comment:
Ah I think it's because you forgot the deps array here.

Reply:
Good catch :fishing_pole_and_fish: ! Fixed everything :tada: